### ISSUE TITLE
Migrate packages from AnthonyMastrean/chocolateypackages

### ISSUE THREAD

**Issue by dgalbraith** [dgalbraith/chocolatey-packages#181]
Anthony Mastrean is looking for package maintainers to take a number of packages forward (see AnthonyMastrean/chocolateypackages/issues/392).

Having agreed to take a number of these packages on they need to be migrated.  Packages for migration from the repository AnthonyMastrean/chocolateypackages are:

- [ ] ant
- [ ] apachedirectorystudio
- [ ] CLOC
- [ ] dejavufonts
- [ ] dependencywalker
- [ ] dotcover-cli
- [x] ~~goggalaxy~~ already migrated to [LinkSantonaka](https://chocolatey.org/profiles/LinkSantonaka)
- [ ] imagej
- [ ] isorecorder
- [ ] isowriter
- [x] ~~Kindle~~ already migrated to [dgalbraith](https://chocolatey.org/profiles/dgalbraith)
- [ ] kindlepreviewer
- [ ] mousewithoutborders
- [ ] myharmony
- [ ] pmd
- [ ] prime95
- [ ] sendtokindle
- [ ] sqljdbc
- [ ] ultramon
- [ ] visioviewer
- [ ] visioviewer2016
- [ ] wudt

**Comment by stadub** [comment]
Hey,
I took over the original `wudt` source maintenance [github repo](https://github.com/stadub/wudt--Windows-USB-DVD-Download-Tool)
so if you will not mind I would like to take chocko package ownership as well

**Comment by dgalbraith** [comment]
Makes sense to consolidate if you are looking after the source maintenance.  If you could advise me on your Chocolatey user name I can add you as a package maintainer.

**Comment by stadub** [comment]
> Makes sense to consolidate if you are looking after the source maintenance. If you could advise me on your Chocolatey user name I can add you as a package maintainer.

https://chocolatey.org/profiles/stadub

**Comment by dgalbraith** [comment]
Added @stadub as a package admin on Chocolatey to facilitate consolidation with the `wudt` tool.
